packages/url/src/filter-url-for-display.js

```diff
@@ -13,6 +13,8 @@
  * @return {string} Displayed URL.
  */
 export function filterURLForDisplay( url, maxLength = null ) {
+	if ( ! url ) return url;
```

Comment:
This function errors if `url` is `undefined` (or rather doesn't have `.replace()` as a method).

This caused issues when composing the functions together in `search-item.js`.

Reply:
Suggestion: Maybe we should move this check into the `getURLForDisplay` and return an empty string early when `url` is `undefined`. This we can keep changes localized without any side-effects.

Reply:
Most of the `url` package methods expect the argument. I'm guessing that's intentional.

Thanks for the following 🙇

Reply:
I mean...I guess if a function is named `filterURLForDisplay` you should probably pass it a URL string 😅